lively.collab/comments/commentBrowser.js

```diff
@@ -6,77 +6,97 @@ import { connect } from 'lively.bindings';
 import { CommentMorph, Badge } from 'lively.collab';
 import { ModeSelector } from 'lively.components/widgets.js';
 
-let instance;
-
 export class CommentBrowser extends Window {
   static close () {
     if (CommentBrowser.isOpen()) {
-      instance.close();
+      CommentBrowser.instance.close();
     }
   }
 
   static get instance () {
-    return instance;
+    return $world.commentBrowser;
   }
 
   static isOpen () {
-    return instance && $world.get('comment browser');
+    return !!(CommentBrowser.instance && !!$world.get('comment browser'));
```

Comment:
are the !! before `$world` really necessary? Because if `CommentBrowser.instance` is not falsy, then `$world.get('comment browser')` should return something different from `undefined`